appveyor.yml

```diff
@@ -25,8 +26,17 @@ install:
 
 build_script:
   - cmd: echo Building on image %APPVEYOR_BUILD_WORKER_IMAGE%, arch %arch%.
-  - cmd: meson setup --backend=ninja build_dir .
-  - cmd: meson compile -v -C build_dir
+  # Build with CMake
+  - cmd: if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" ( SET "VS_VERSION=Visual Studio 15 Win64" )  # 15 because Visual Studio 2017 is actually version 15.0. Win64 because it is on x64.
+  - cmd: if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2019" ( SET "VS_VERSION=Visual Studio 16" )  # 16 because Visual Studio 2019 is actually version 16.0.
```

Comment:
This will always build the 64-bit version using MSVC2017, and 32-bit (I think) on MSVC2019.
See https://cmake.org/cmake/help/v3.19/generator/Visual%20Studio%2015%202017.html#platform-selection on how to set the architecture.
You can add environment variables to the matrix (in addition to `arch` and use them here)

Reply:
Thanks, it was easy to do it with the link you provided! Here it is: https://github.com/Martinsos/edlib/pull/167/commits/ac0ff7177e86ab9ddfdc916122f3cbe87140bea8, this should fix it.